src/content/advanced/configure-dex-in-your-cluster/index.md

```diff
@@ -193,9 +176,46 @@ data:
             redirectURI: https://dex.<CLUSTERID>.<BASEDOMAIN>/callback
 ```
 
+{{< /tab >}}
+{{< tab title="Okta">}}
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dex-app-user-values
+  namespace: <CLUSTERID>
+data:
+  values: |
+    isWorkloadCluster: true
+    oidc:
+      expiry:
+        signingKeys: 6h
+        idTokens: 30m
+      customer:
+        enabled: true
+        connectors:
+        - id: customer
+          connectorName: test
+          connectorType: oidc
+          connectorConfig: >-
+            clientID: <CLIENT-ID-SET-IN-YOUR-IdP>
+            clientSecret: <CLIENT-SECRET-SET-IN--YOUR-IdP>
+            insecureEnableGroups: true
+            getUserInfo: true
+            scopes:
+            - email
+            - groups
+            - profile
+            issuer: https://<OKTA_OIDC_ENDPOINT>
+            redirectURI: https://dex.<CLUSTERID>.<BASEDOMAIN>/callback
+```
+
 {{< /tab >}}
 {{< /tabs >}}
 
+__Warning__: With `oidc` connector you might need to add `getUserInfo` in the connector configuration to force a second called to IdP in order to get groups (for example required by Okta). More info [here](https://github.com/dexidp/dex/issues/1065).
```

Comment:
```suggestion
__Warning__: With `oidc` connector you might need to add `getUserInfo` in the connector configuration to force a second call to the identity provider in order to get groups. This is required for example by Okta. More info on this can be found in [dexipd/dex#1065](https://github.com/dexidp/dex/issues/1065).
```